src/entity_extraction/training/spacy_ner/create_config.py

```diff
@@ -0,0 +1,40 @@
+# Author: Jenit Jain
+# Date: 2023-06-21
+# Inspired from https://github.com/explosion/projects/blob/v3/pipelines/ner_demo_update/scripts/create_config.py
+"""This script create a config file when resuming training of a spacy model from a past checkpoint
+
+Usage: create_config.py --model_path=<model_path> --output_path=<output_path>
+
+Options:
+    --model_path=<model_path>         The path to the model artifacts.
+    --output_path=<output_path>       The path to the output config file.
+"""
+
+from pathlib import Path
+from docopt import docopt
+import spacy
+
+opt = docopt(__doc__)
+
+def create_config():
```

Comment:
nitpick: Missing docstring, why is this function creating a config? For what? It is stated in the docopt section but any VS code function previews I don't think pull in docopt descriptions unfortunately